api/src/main/java/com/codeforcommunity/dto/site/EditStewardshipRequest.java

```diff
@@ -0,0 +1,30 @@
+package com.codeforcommunity.dto.site;
+
+import com.codeforcommunity.exceptions.HandledException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class EditStewardshipRequest extends RecordStewardshipRequest {
+
+  public EditStewardshipRequest(
+      java.sql.Date date, boolean watered, boolean mulched, boolean cleaned, boolean weeded) {
+    super(date, watered, mulched, cleaned, weeded);
+  }
+
+  private EditStewardshipRequest() {}
+
+  @Override
+  public List<String> validateFields(String fieldPrefix) throws HandledException {
+    String fieldName = fieldPrefix + "edit_stewardship_request.";
+    List<String> fields = new ArrayList<>();
+
+    if (getDate() == null) {
```

Comment:
🔧 Can we access the `date` field directly instead of using a getter so that it's consistent with the rest of the method (and other `validateFields` methods)? Also, since the field validation logic is the same for `RecordStewardshipRequest` and `EditStewardshipRequest`, we can abstract it out into a helper method.